fireatlas/postprocess.py

```diff
@@ -265,6 +288,18 @@ def save_fire_layers(allfires_gdf_fid, region, fid, tst):
 
         data.to_file(os.path.join(output_dir, f"{layer}.fgb"), driver="FlatGeobuf")
```

Comment:
Does this need to be `lf_{layer}.fgb` as well?

Reply:
good point, let's ignore this for now b/c we are not ingesting it since `nplist` is many to an FID and we don't know how to create a primarykey off it (we decided to ignore it in legacy code as well)